src/popupmenu.c

```diff
@@ -628,6 +631,8 @@ pum_redraw(void)
 	{
 	    hlf = hlfs[round];
 	    attr = highlight_attr[hlf];
+	    if (pum_array[idx].pum_extrahlattr > 0)
+		attr = hl_combine_attr(attr, pum_array[idx].pum_extrahlattr);
```

Comment:
The priority of this highlight group is inconsistent. Here it has higher priority than the default ones, but in `pum_screen_puts_with_attrs()` it has lower priority. Also, I wonder if the combination should be done in `pum_compute_text_attrs()` instead.

Reply:
Oops, forgot to change this. Because  can't test ceterm gui properties in github ci. So I adjusted the priority. Use a fake color simulation.

> Also, I wonder if the combination should be done in pum_compute_text_attrs() instead.

this function invoke on attrs not null right?